test/document_stores/test_elasticsearch.py

```diff
@@ -348,6 +357,34 @@ def test_get_document_by_id_excluded_meta_data_has_no_influence(self, mocked_doc
     @pytest.mark.unit
     def test_write_documents_req_for_each_batch(self, mocked_document_store, documents):
         mocked_document_store.batch_size = 2
-        with patch("haystack.document_stores.elasticsearch.bulk") as mocked_bulk:
+        with patch("haystack.document_stores.elasticsearch7.bulk") as mocked_bulk:
             mocked_document_store.write_documents(documents)
             assert mocked_bulk.call_count == 5
+
+    @pytest.mark.unit
+    @pytest.mark.skipif(ES_VERSION[0] != 7, reason="Only run for ES7")
+    def test_import_from_haystack_document_stores_es7(self):
+        from haystack.document_stores import ElasticsearchDocumentStore
+
+        assert ElasticsearchDocumentStore.__module__ == "haystack.document_stores.elasticsearch7"
```

Comment:
Why is this not tested in `test_elasticsearch7.py`?

Reply:
True, I'll move this.